### PR TITLE
Fix pairwise_judge to use A/B instead of col_A/col_B. col_A/col_B use 2 tokens, so cascade fails

### PR DESCRIPTION
**Pull request by harshitgupta412**
## Essential Elements of an Effective PR Description Checklist
- [ ] The purpose of the PR, such as "Fix some issue (link existing issues this PR will resolve)".
- [ ] The test plan, such as providing test command.
- [ ] The test results, such as pasting the results comparison before and after, or e2e results
- [ ] (Optional) The necessary documentation update, such as updating `README.md` and `examples` for new features.

PLEASE FILL IN THE PR DESCRIPTION HERE ENSURING ALL CHECKLIST ITEMS ABOVE HAVE BEEN CONSIDERED.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Refactoring (no functional changes)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, updating docstrings
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

**BEFORE SUBMITTING, PLEASE READ <https://github.com/lotus-data/lotus/blob/main/CONTRIBUTING.md>** 
anything written below this line will be removed by GitHub Actions

### CODE REVIEW

**Review by liana313 — COMMENTED** [review]
Let's also update CI tests to include the pairwise judge and test for edge cases on this

File: .github/tests/lm_tests.py [hunk 1]
```diff
@@ -712,3 +712,23 @@ def test_pairwise_judge(setup_models, model):
     )
     assert list(df["_judge_0"].values) == ["A", "B"]
     assert list(df["_judge_1"].values) == ["A", "B"]
+
+
+@pytest.mark.parametrize("model", get_enabled("gpt-4o-mini"))
```
Comment (liana313) [inline .github/tests/lm_tests.py:717]: This is a start, but we need more than one CI test for cascade testing. We should cover edge cases (e.g., when there's an existing col A and col B)
